packages/react-native-tab-view/src/TabView.tsx

```diff
@@ -24,6 +25,7 @@ import type {
 
 export type Props<T extends Route> = Omit<PagerProps, 'layoutDirection'> & {
   onIndexChange: (index: number) => void;
+  onPageSelected?: NativeProps['onPageSelected'];
```

Comment:
Rename `onPageSelected` to `onTabSelect` to match the terminology of the library